application-admintools-ui/src/main/resources/AdminTools/WebHome.xml

```diff
@@ -47,7 +47,11 @@
 {{velocity}}
 #set ($mainReference = $services.model.createDocumentReference('', ['AdminTools', 'Code'], 'ConfigurationClass'))
 #if ($services.licensing.licensor.hasLicensureForEntity($mainReference))
-  {{dashboard/}}
+  #if ($isAdvancedUser || $isSuperAdmin || $hasAdmin)
+    {{dashboard/}}
+  #else
+    {{error}}$escapetool.xml($services.localization.render('adminTools.dashboard.missingRights')){{/error}}
```

Comment:
theres no need to `escape.xml` because you are are not inside an {{html}} macro. You should escape for xwiki syntax. https://github.com/xwiki/xwiki-platform/blob/HEAD/xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminTemplatesSheet.xml#L115-L116